.github/workflows/build.yml

```diff
@@ -203,7 +203,7 @@ jobs:
     env:
       USE_NINJA: "1"
       WASM2C_CC: "clang"
-      WASM2C_CFLAGS: "-DWASM_RT_USE_MMAP=1 -DWASM_RT_SKIP_SIGNAL_RECOVERY=1 -DWASM_RT_NONCONFORMING_UNCHECKED_STACK_EXHAUSTION=1 -DWASM2C_TEST_EMBEDDER_SIGNAL_HANDLING -DWASM_RT_ALLOW_SEGUE=1 -mfsgsbase -DWASM_RT_SANITY_CHECKS=1 -Wno-pass-failed"
+      WASM2C_CFLAGS: "-DWASM_RT_USE_MMAP=1 -DWASM_RT_SKIP_SIGNAL_RECOVERY=1 -DWASM_RT_NONCONFORMING_UNCHECKED_STACK_EXHAUSTION=1 -DWASM2C_TEST_EMBEDDER_SIGNAL_HANDLING -DWASM_RT_ALLOW_SEGUE=1 -DWASM_RT_SEGUE_FREE_SEGMENT=1 -mfsgsbase -DWASM_RT_SANITY_CHECKS=1 -Wno-pass-failed"
```

Comment:
On the name of this new macro, would it make sense to call it something like `WASM_RT_SEGUE_PERSISTENT` ?

Reply:
Having read the change again, I'm not sure my name is any better.  lgtm either way.

Reply:
Ah ok, if no preference, I'll leave it as is.

Reply:
(hmm, what about `SEGUE_SEGMENT_EXCLUSIVE`?)